<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconButtonSettings</class>
 <widget class="QWidget" name="IconButtonSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="4" column="0">
    <spacer name="hspL">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <spacer name="hspR">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <spacer name="vspBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <spacer name="vspSeparator">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="MaterialIconButton" name="iconBtn">
     <property name="icon">
      <iconset resource="../../resources/library/qtmaterialwidgets.qrc">
       <normaloff>:/icons/action/svg/production/ic_account_balance_24px.svg</normaloff>:/icons/action/svg/production/ic_account_balance_24px.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QSpinBox" name="sizeSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QCheckBox" name="disabledCheckBox">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialIconButton</class>
   <extends>QAbstractButton</extends>
   <header location="global">QtMaterialWidgets/iconbutton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/library/qtmaterialwidgets.qrc"/>
 </resources>
 <connections/>
</ui>
